pymedphys/_dicom/anonymise/strategy.py

```diff
@@ -60,9 +60,13 @@ def _get_vr_anonymous_hardcode_replacement_value(current_value, value_representa
 _keys = list(VR_TO_REPLACEMENT_MAP.keys())
 _keys.append("SQ")
 
-ANONYMISATION_HARDCODE_DISPATCH = {
-    key: functools.partial(
-        _get_vr_anonymous_hardcode_replacement_value, value_representation=key
-    )
-    for key in _keys
-}
+
+@functools.lru_cache(maxsize=1)
+def get_default_hardcode_dispatch():
```

Comment:
I was actually hoping to remove the need for this wrapping entirely. I'll show you what I mean in the next commit.